variables.tf

```diff
@@ -1,78 +1,102 @@
-variable subscriptionId {
+variable "subscriptionId" {
   type        = string
   description = "The ID of the Azure subscription to be used."
 }
 
-variable environment {
+variable "environment" {
   type        = string
   description = "The Azure environment to be used."
 }
 
-variable location {
+variable "location" {
   type        = string
   description = "The Azure location to be used."
 }
 
-variable infrastructurename {
+variable "infrastructurename" {
   type        = string
   description = "The name of the infrastructure. e.g. simphera-infra"
 }
 
-variable tags {
-  type        = map
+variable "tags" {
+  type        = map(any)
   description = "The tags to be added to all resources."
   default     = {}
 }
 
-variable linuxNodeSize {
+variable "linuxNodeSize" {
   type        = string
   description = "The machine size of the Linux nodes for the regular services"
   default     = "Standard_D4s_v3"
```

Comment:
Currently, we already use "v4". However, there is already "v5" available.

Reply:
Fixed to v4 in 5669c7b18fa3e61302870da1d8caf3aa42de5316